docs/data/material/discover-more/related-projects/related-projects.md

```diff
@@ -7,63 +7,74 @@ Developers from the community have built some excellent supplemental tools for w
 Do you have a project that you think should be featured here?
 Feel free to submit a pull request and the maintainers will work with you to write the description.
 
-## Premium tools
+## App starters
 
-- [ScaffoldHub](https://v2.scaffoldhub.io/scaffolds/react-material-ui?partner=1):
+### Admin frameworks
+
+- [React-admin](https://github.com/marmelab/react-admin): A front-end framework for building data-driven applications running in the browser on top of REST/GraphQL APIs.
+- [Refine](https://github.com/refinedev/refine): An open source, headless React meta-framework for the rapid development of web applications.
+
+### Paid starters
+
+- [Divjoy](https://divjoy.com?via=material-ui):
+  - Create a Material UI app in minutes.
+  - Templates, authentication, database integration, subscription payments, and more.
 
+- [ScaffoldHub](https://v2.scaffoldhub.io/scaffolds/react-material-ui?partner=1):
   - Tool for building web applications.
   - Choose your framework and library (React with Material UI).
   - Choose your database (SQL, MongoDB or Firestore).
   - Model your database and application with the intuitive GUI.
   - Generate your application, including a complete scaffolded backend.
   - Preview your application online and download the generated code.
 
-- [Divjoy](https://divjoy.com?via=material-ui):
-
-  - Create a Material UI app in minutes.
-  - Templates, authentication, database integration, subscription payments, and more.
-
-## Free tools
-
-### Admin frameworks
-
-- [React-admin](https://github.com/marmelab/react-admin): A front-end framework for building data-driven applications running in the browser on top of REST/GraphQL APIs.
-- [Refine](https://github.com/refinedev/refine): An open source, headless React meta-framework for the rapid development of web applications.
-
-### Theming tools
+## Theming tools
 
 - [Material palette generator](https://m2.material.io/inline-tools/color/): The official Material Design 2 palette generator tool.
 
-### Supplementary components
+## Complementary components
 
-#### Layout
+### Layout
```

Comment:
So we can anchor links them in the search. If I search for rich text editor in Algolia, I want to land on the header. Also so this can be shared with teammates, and so we can skim read with the TOC.